canonical-web-and-design/maas-ui
# feat(ui): add project tab to LXD KVM details page

## Done

- Added a tab for "Project" to the LXD details page
- Renamed `KVMSummary` to `KVMResources` to match the tab name, plus "summary" is a bit vague/different between virsh and lxd pods.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Click on a LXD KVM from the KVM list and check that you are taken to the projects tab
- Click on a virsh KVM from the KVM list and check that you are taken to the resources tab instead
- Check that manually navigating to "/MAAS/r/kvm/<id>/projects" for a virsh KVM will redirect you to the resources tab

## Fixes

Fixes #2229
